go.mod

```diff
@@ -11,13 +11,13 @@ require (
 	github.com/BurntSushi/toml v1.0.0
 	github.com/DataDog/opencensus-go-exporter-datadog v0.0.0-20180917103902-e6c7f767dc57
 	github.com/aws/aws-sdk-go v1.32.7
-	github.com/bsm/redislock v0.4.2
+	github.com/bsm/redislock v0.8.0
 	github.com/fatih/color v1.13.0
-	github.com/go-redis/redis/v7 v7.4.1
+	github.com/go-redis/redis/v9 v9.0.0-beta.2
```

Comment:
Is it safe to use a beta version of the go-redis library? I assume its coming from bsm/redislock. I think the real question here is to understand if we only depend on bsm/redislock for our redis needs or we rely on go-redis/redis directly as well.

alternatively, can we check if we really need to use v0.8.0 or some version before that so that we depend on stable release of go-redis/redis?

Reply:
Good catch! The required functionality exists in `bsm/redislock@v0.7.2` onwards which uses a stable go-redis release so should be possible for us to use that.

I'll test out my changes with v0.7.2 (since that's the next one before v0.8.0) and update the PR.

Reply:
Updated, `v0.7.2` is also working fine. Reverted the test changes as well since it's not required anymore.